ember-server/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -40,11 +40,10 @@ import scodec.bits.ByteVector
 
 private[server] object ServerHelpers {
 
-  private val closeCi = "close".ci
-
-  private val connectionCi = "connection".ci
-  private val close = Connection(NonEmptyList.of(closeCi))
-  private val keepAlive = Connection(NonEmptyList.one("keep-alive".ci))
+  private[this] val closeCi = "close".ci
+  private[this] val connectionCi = "connection".ci
+  private[this] val close = Connection(NonEmptyList.of(closeCi))
+  private[this] val keepAlive = Connection(NonEmptyList.one("keep-alive".ci))
```

Comment:
I got shouted down on this years ago, but I'm still of the opinion these header constants belong on the header companions.